src/styles/main.scss

```
@@ -1,3 +1,44 @@
+@import "./utils/reset.scss";
+@import "./utils/variables.scss";
+
 body {
-  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+.box {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, calc(100% / 3)));
+
+  &__item {
+    position: relative;
+    height: 300px;
+
+    @each $color, $index in $colors {
+      &:nth-child(#{$index}) {
+        background-color: $color;
+
+        &::before {
+          position: absolute;
+          top: 50%;
+          left: 50%;
+          transform: translate(-50%, -50%);
+          content: "#{$index}";
+        }
+      }
+    }
+  }
+}
+
+@media (max-width: 900px) {
+  .box {
+    grid-template-columns: repeat(auto-fill, minmax(300px, calc(100% / 2)));
+  }
+}
+
+@media (max-width: 600px) {
+  .box {
+    grid-template-columns: repeat(auto-fill, minmax(300px, 100%));
+  }
```

це можна написати в селекторі box
```suggestion
@media (max-width: 900px) {
  grid-template-columns: repeat(auto-fill, minmax(300px, calc(100% / 2)));
}

@media (max-width: 600px) {
  grid-template-columns: repeat(auto-fill, minmax(300px, 100%));
}
```

---

agree